var/spack/repos/builtin/packages/transabyss/package.py

```diff
@@ -0,0 +1,46 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Transabyss(Package):
+    """De novo assembly of RNAseq data using ABySS"""
+
+    homepage = "http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss"
+    url      = "http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss/releases/1.5.5/transabyss-1.5.5.zip"
+
+    version('1.5.5', '9ebe0394243006f167135cac4df9bee6')
+
+    depends_on('abyss@1.5.2')
+    depends_on('python@2.7.6:', type=('build', 'run'))
+    depends_on('py-igraph@0.7.0:')
+    depends_on('blat')
+
+    def install(self, spec, prefix):
+        mkdirp(prefix.bin)
+        install('transabyss', prefix.bin)
+        install('transabyss-merge', prefix.bin)
+        install_tree('bin', prefix.bin.bin)
```

Comment:
There's a certain way that the package structure has to be set up in order for transabyss to work--the executables have to be on the same level as bin and utilites. So I wasn't sure whether it would be better practice to throw the executables in bin and have the other two directories available under prefix/bin/bin and prefix/bin/utilities, or to have the two executables just installed to prefix not inside of a directory and have bin and utilities be prefix/bin and prefix/utilities, respectively